packages/hardhat/contracts/periphery/ControllerHelper.sol

```diff
@@ -665,11 +664,6 @@ contract ControllerHelper is UniswapControllerHelper, EulerControllerHelper, IER
                         data[i].data,
                         (ControllerHelperDataType.GeneralSwapParams)
                     );
-
-                    // make sure not to fail
-                    uint256 currentBalance = IERC20(swapParams.tokenIn).balanceOf(address(this));
-                    if (currentBalance < swapParams.amountIn) swapParams.amountIn = currentBalance;
-
                     _exactInFlashSwap(
```

Comment:
Why this removed?

Reply:
this came in from rebase I think. It's changed in main now

Reply:
added back